pkgs/servers/trezord/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchgit, curl, cmake, boost, gcc5, protobuf, pkgconfig, jsoncpp
+, libusb1, libmicrohttpd
+}:
+
+let
+  version = "1.2.0";
+in
+
+stdenv.mkDerivation rec {
+  name = "trezord-${version}";
+
+  src = fetchgit {
```

Comment:
Okay, I tried using `fetchFromGitHub` but for some reason it was grabbing a different version with a different hash which for some reason didn't build. So I've gone back to `fetchgit`.